libdeno/BUILD.gn

```diff
@@ -16,21 +13,27 @@ config("deno_config") {
   }
 }
 
-v8_static_library("v8") {
-  public_deps = [
-    "//build/win:default_exe_manifest",
+v8_component("v8") {
+  configs = [ ":deno_config" ]
+  deps = [
     "//third_party/v8:v8",
     "//third_party/v8:v8_libbase",
     "//third_party/v8:v8_libplatform",
     "//third_party/v8:v8_libsampler",
+
+    # TODO: do this in Cargo builds only?
   ]
-  configs = [ ":deno_config" ]
+
+  if (!is_win) {
+    deps += [ "//buildtools/third_party/libc++:libc++" ]
+  }
```

Comment:
It's kinda strange this is only on V8. Shouldn't libdeno need this too?

Reply:
Yeah it should probably move to a config.

Reply:
I'm guessing there actually is an external config for it  `//buildtools/third_party/libc++:external_config` or something.